apps/console/src/features/applications/components/settings/attribute-management/attribute-list-item.tsx

```diff
@@ -49,6 +49,14 @@ interface AttributeListItemPropInterface extends TestableComponentInterface {
      * Make the form read only.
      */
     readOnly?: boolean;
+    /**
+     * Add a label.
+     */
+    label?: any;
```

Comment:
Try to avoid `any` whereever possible.

`ReactNode` supports both `strings` and react elements.
```suggestion
    label?: ReactNode;
```